Source/Shaders/BillboardCollectionVS.glsl

```diff
@@ -186,38 +224,19 @@ void main()
     float pixelOffsetScale = getNearFarScalar(pixelOffsetScaleByDistance, lengthSq);
     pixelOffset *= pixelOffsetScale;
 #endif
-
-    vec4 positionWC = czm_eyeToWindowCoordinates(positionEC);
-    
-    vec2 halfSize = imageSize * scale * czm_resolutionScale;
-    halfSize *= ((direction * 2.0) - 1.0);
-    
-    positionWC.xy += (origin * abs(halfSize));
     
-#if defined(ROTATION) || defined(ALIGNED_AXIS)
-    if (!all(equal(alignedAxis, vec3(0.0))) || rotation != 0.0)
+#ifdef TEST_GLOBE_DEPTH
+    vec4 offsetPosition = positionEC + vec4(0.0, 0.0, -positionEC.z * 0.05, 0.0);
+    vec4 wc = computePositionWindowCoordinates(offsetPosition, vec2(0.0, 0.0), scale, direction, origin, vec2(0.0), pixelOffset, alignedAxis, rotation);
+    float d = texture2D(czm_globeDepthTexture, wc.xy / czm_viewport.zw).r;
```

Comment:
What if number of vertex shader texture units is zero or depth textures are not supported?  We don't need to throw an exception, but there should be a trivial way for the user to see if clamp-to-ground is supported and perhaps we should warn in the console if they ask if it and the system doesn't support it.